common/config/azure-pipelines/jobs/ci-core.yaml

```diff
@@ -1,65 +1,39 @@
 parameters:
-  name: ''
-  pool: ''
-  nodeVersion: ''
-  buildIos: false
-
-jobs:
-- job:
-  strategy:
-    matrix:
-      "Windows_${{ parameters.name }}":
-        OS: windows-latest
-        platform: Windows_NT
-        name: $(win_pool)
-      "Linux_${{ parameters.name }}":
-        OS: ubuntu-latest
-        platform: Linux
-        name: $(linux_pool)
-      "MacOS_${{ parameters.name }}":
-        OS: macOS-latest
-        platform: Darwin
-        name: $(mac_pool)
-
-  timeoutInMinutes: 120
-
-  pool: ${{ parameters.pool }}
-
-  workspace:
-    clean: all
-
-  steps:
-    - checkout: self
-      clean: true
-    - task: NodeTool@0
-      displayName: "Use Node ${{ parameters.nodeVersion }}"
-      inputs:
-        versionSpec: ${{ parameters.nodeVersion }}
-        checkLatest: true
-
-    - script: |
-        git config --local user.email imodeljs-admin@users.noreply.github.com
-        git config --local user.name imodeljs-admin
-      displayName: git config
-
-    - powershell: |
-        # Get the new version number.
-        $json = Get-Content -Raw -Path common/config/rush/version-policies.json | ConvertFrom-Json
-        $currVersion = $json[0].version
-
-        $newBuildNumber = $currVersion + "_$(Build.BuildNumber)"
-
-        Write-Host "##vso[build.updatebuildnumber]$newBuildNumber"
-      displayName: Set build number
-      condition: and(succeeded(), eq(variables['Agent.OS'], 'Windows_NT'))
+- name: nodeVersion
```

Comment:
With the way the pipelines are laid out now. This template doesn't seem necessary anymore and can probably be deleted.

Reply:
Yeah, it shouldn't be used anywhere else. Forgot to delete. Thanks